.drone.yml

```diff
@@ -40,7 +41,11 @@ pipeline:
     volumes:
       - /cache:/cache
     commands:
-      - composer install --ansi --no-suggest --no-progress
+      - composer self-update --2
+      - composer install --ansi --no-progress
+    when:
+      matrix:
+        COMPOSER_BOUNDARY: highest
```

Comment:
Would be nice also explicitly declare that this pipeline for highest like here: https://github.com/openeuropa/oe_contact_forms/pull/44/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R46